src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -2,18 +2,26 @@
 
 import com.google.gson.Gson;
 import com.training.basetest.ApiBaseTest;
-import com.training.pojos.ga.authentication.Payload;
 import com.training.pojos.ga.validation.Request;
 import com.training.utilities.RestEngine;
 import io.restassured.RestAssured;
+import org.apache.commons.lang3.RandomStringUtils;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class GuestAccountTests extends ApiBaseTest {
     private final String EMAIL = "email@email.com";
+    private final String PASSWORD = "password1";
```

Comment:
Since this is not static, this should not be in upper case.
Only static & final variables should be in upper case